ubuntu_bug_triage/triage.py

```diff
@@ -98,12 +99,21 @@ def updated_bugs(self):
         for bug_id in sorted(self._tasks_to_bug_ids(updated_tasks)):
             bug = Bug(self.launchpad.bugs[bug_id])
 
+            lpbug = self.launchpad.bugs[bug_id]
+            commenter = None
+            for comment in lpbug.messages:
+                commenter = comment.owner_link.split('~')[1]
+            person = None
+            for active in lpbug.activity:
+                person = active.person_link.split('~')[1]
+
```

Comment:
Rather than calling `launchpad.bugs()` twice, can we move this logic to find the commenter and person into the `Bug` class found in `bug.py` so both `TeamTriage` and `PackageTriage` can rely on it?

I think adding a property like `last_commenter` and `last_active_user` that could return these would work. Thoughts?

Reply:
I've done those changes now but ended up having only one 'last_active_user' because we want to check what was the most recent changes on the bug and it doesn't matter if that's an edit or comment